src/condor_ce_jobmetrics

```diff
@@ -37,23 +37,28 @@ def secure_json_write(fname, contents):
     finally:
         temp_file.close()
 
+
+_newstat = lambda: {"Running": 0, "Idle": 0, "Held": 0, "Jobs": 0}
+
 def process_one_schedd(ad):
     import htcondor
     import classad
     ad = classad.ClassAd(ad)
     schedd = htcondor.Schedd(ad)
     status_map = {1: "Idle", 2: "Running", 5: "Held"}
-    _newstat = lambda: {"Running": 0, "Idle": 0, "Held": 0, "Jobs": 0}
     total_results = _newstat()
+    gpu_total_results = _newstat()
     vo_results = collections.defaultdict(_newstat)
     job_count = collections.defaultdict(_newstat)
+    gpu_job_count = collections.defaultdict(_newstat)
     print "Processing CE %s." % ad.get("Name", "Unknown")
     try:
         for job in schedd.xquery("true", ["JobStatus", 'x509UserProxyVOName',
-                                          'x509UserProxyFirstFQAN', 'x509userproxysubject']):
+              'x509UserProxyFirstFQAN', 'x509userproxysubject', 'RequestGPUs']):
```

Comment:
```suggestion
                                          'x509UserProxyFirstFQAN', 'x509userproxysubject', 'RequestGPUs']):
```

It'd be nice to retain the spacing here for clarity

Reply:
B-but... my poor 80-wide terminal...